Detectors/GlobalTrackingWorkflow/study/src/SVStudy.cxx

```diff
@@ -211,13 +216,24 @@ o2::dataformats::V0Ext SVStudySpec::processV0(int iv, o2::globaltracking::RecoCo
       }
       if (gidset[GTrackID::ITSTPC].isSourceSet()) {
         auto mtc = recoData.getTPCITSTrack(gidset[GTrackID::ITSTPC]);
-        lb = recoData.getTrackMCLabel(gidset[GTrackID::ITSTPC]);
+        lb[ip] = recoData.getTrackMCLabel(gidset[GTrackID::ITSTPC]);
         prInfo.chi2ITSTPC = mtc.getChi2Match();
-        if (lb.isValid()) {
-          prInfo.corrITSTPC = !lb.isFake();
+        if (lb[ip].isValid()) {
+          prInfo.corrITSTPC = !lb[ip].isFake();
         }
       }
     }
+    if (mUseMC) { // temp store of mctrks
+      mcTrks[ip] = mcReader->getTrack(lb[ip]);
```

Comment:
You should check if `lb[ip].isValid()`: in principle, tracks could be made entirely from the noise.

Reply:
ah right, thanks!